OXRS-SHA-StateMonitor-ESP32-FW.ino

```diff
@@ -100,23 +99,25 @@ void setup()
   // initialize screen
   screen.begin();
   
-  // Speed up I2C clock for faster scan rate (after bus scan and screen init)
+  // Speed up I2C clock for faster scan rate (after bus scan)
   Wire.setClock(I2C_CLOCK_SPEED);
 
-  // Display the firmware version and initialise the port display
-  screen.draw_logo(FW_VERSION);
+  // Display the header and initialise the port display
+  screen.draw_header(FW_MAKER_CODE, FW_SHORT_NAME, FW_VERSION, FW_PLATFORM);
   screen.draw_ports(g_mcps_found);
 
   // Set up ethernet and obtain an IP address
   byte mac[6];
   initialiseEthernet(mac);
+  
+  // Display IP and MAC addresses on screen
+  screen.show_ethernet();

```

Comment:
This can be removed based on my suggested changes I think?

Reply:
done